src/index.js

```diff
@@ -521,7 +521,11 @@ function honoWrapper(compiler, options) {
     /**
      * @returns {string | undefined}
      */
-    req.getURL = () => c.req.url;
+    req.getURL = () => {
+      // @ts-ignore
+      const url = c.get("c.req.url");
+      return url || c.req.url;
```

Comment:
Can you describe this change and why?

Reply:
https://github.com/bripkens/connect-history-api-fallback/blob/74564d61c2db06e302923a23bd459bb5dd91712b/lib/index.js#L65

Reply:
ok